package.json

```diff
@@ -50,5 +52,44 @@
     "bugs": {
         "url": "https://github.com/Microsoft/axe-sarif-converter/issues"
     },
-    "homepage": "https://github.com/Microsoft/axe-sarif-converter#readme"
+    "homepage": "https://github.com/Microsoft/axe-sarif-converter#readme",
+    "license-check-and-add-config": {
+        "folder": "./",
+        "license": "copyright-header.txt",
+        "exact_paths_method": "EXCLUDE",
+        "exact_paths": [
+            "./.vscode",
+            "./.git",
+            "./.github",
+            "./node_modules",
+            "./copyright-header.txt"
+        ],
+        "file_type_method": "INCLUDE",
+        "file_types": [
+            ".ts",
+            ".d.ts",
```

Comment:
If we omit this does it not fall under the .ts rule?

Reply:
This part cannot be omitted because it tells license-check-and-add which files it should look at.